file_watcher_operator/file_watcher_operator.py

```diff
@@ -243,12 +319,13 @@ def deploy_pv(pv_spec: Mapping[str, Any], name: str, children: list[Any]) -> Non
     :param children: The operators children
     :return: None
     """
-    core_api = kubernetes.client.CoreV1Api()
+    core_api = client.CoreV1Api()
     # Check if PV exists else deploy a new one
-    if pv_spec["metadata"]["name"] not in [ii.metadata.name for ii in core_api.list_persistent_volume().items]:
+    if pv_spec.metadata.name not in [ii.metadata.name for ii in core_api.list_persistent_volume().items]:
         logger.info("Starting deployment of PV: filewatcher-%s", name)
         pv = core_api.create_persistent_volume(body=pv_spec)
-        children.append(pv.metadata.uid)
+        if children is not None:
```

Comment:
```suggestion
        if children:
```

Reply:
Given python is a dynamic typed language and I want to explicitly check if it's None, not just if a value is, 0, False, or None. Is it likely that they are the same? Sure. Will that always be the case in the long term? I do not know, hence the explicit.